andela/wger_akira
# Ft exercise information #142465439

#### What does this PR do
Add an API end point for all information concerning an exercise.
#### Description of Task to be completed?
There should be a new API endpoint that gives all information about an exercise to make it easier for clients to get all that information since only one request needs to be done.
####  How should this be manually tested?
Run the application and  navigate to the browse API page and check the sample information from the exerciseinformation link.
#### What are the necessary pivotal tracker stories
#142465439
